src/Annotation/KeyType.php

```diff
@@ -41,4 +41,13 @@ class KeyType extends Plugin {
    */
   public $description;
 
+  /**
+   * The storage method of the key type.
+   *
+   * This is an enumeration of {file, config, database, upload}
```

Comment:
We may want to give short descriptions of each method, either here or in the README